apis/v1alpha1/gatewayclass_types.go

```diff
@@ -62,24 +62,24 @@ type GatewayClassSpec struct {
 	// +required
 	Controller string `json:"controller"`
 
-	// AllowedGatewayNamespaceSelector is a selector of namespaces that Gateways
-	// can use this GatewayClass from. This is a standard Kubernetes
-	// LabelSelector. Controllers must not support Gateways in namespaces
-	// outside this selector.
+	// AllowedGatewayNamespaces is a selector of namespaces that Gateways of
+	// this class can be created in. Implementations must not support Gateways
+	// when they are created in namespaces not specified by this field.
 	//
-	// An empty selector (default) indicates that Gateways can use this
-	// GatewayClass from any namespace.
+	// Gateways that appear in namespaces not specified by this field must
+	// continue to be supported if they have already been provisioned. This must
+	// be indicated by the Gateway's presence in the ProvisionedGateways list in
+	// the status for this GatewayClass. If the status on a Gateway indicates
+	// that it has been provisioned but the Gateway does not appear in the
+	// ProvisionedGateways list on GatewayClass it must not be supported.
 	//
-	// When a Gateway attempts to use this class from a namespace that is not
-	// allowed by this selector, the controller implementing the GatewayClass
-	// may add a new "ForbiddenNamespaceForClass" condition to the Gateway
-	// status. Adding this condition is considered optional since not all
-	// controllers will have access to all namespaces.
+	// When this field is unspecified or an empty selector, Gateways will be
+	// able to use this GatewayClass in any namespace.
```

Comment:
```suggestion
	// able to use this GatewayClass in any namespace (default).
```

Reply:
Thanks - updated the wording here to be more clear about the default behavior.